tests/kernel_performance/login_console.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary:  ssh login
+# Maintainer: Joyce Na <jna@suse.de>
+
+package login_console;
+use base "y2logsstep";
+use strict;
+use warnings;
+use File::Basename;
+use testapi;
+use ipmi_backend_utils;
+
+sub login_to_console {
```

Comment:
This seems to be very similar to tests/virt_autotest/login_console.pm , maybe you can just use that test module?

Reply:
Yes,after do some test and it passed,so remove this file and reuse tests/virt_autotest/login_console.pm
http://10.67.132.138/tests/2021